docusaurus/docs/iOS/client/state-layer/state-layer-overview.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Structured Concurrency State Layer
+---
+
+## Introduction
+
+The `StreamChat` framework represents the Stream API with controllers which use completion handlers and delegates for communicating about state changes. In addition to controllers, we provide an alternative way of interacting with the Stream API. The alternative state layer embraces structured concurrency and provides an interface of using async functions.
```

Comment:
Probably good to mention that the new objects are use-case / interactor based, where you can perform different operations on them. Good opportunity to say that it blends well with our video SDK, which can be linked as well. Structured concurrency is more of a technical term.

Reply:
Makes sense, I'll reword this part.